designs/virtualization/4.5/Scheduling/Firmware-config/Firmware-config.md

```diff
@@ -0,0 +1,25 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# Firmware Config
+
+Exposing Firmware Config for VM.
+
+![VM page](img/Firmware-0-0.jpg)
+Firmware Config is placed in the Scheduling sction of the VM details tab.
```

Comment:
Should we explain how this is related to scheduling?

Reply:
I think it is not really related to scheduling.
@jtomasek @glekner please correct me if I'm wrong:
It came to my desk as a part of a bundle of features which we call "Fine-grained CPU features"
Some, if not most of these features affect the scheduling as they require a type or size of resources from the host. But this particular feature is an internal VM configuration, unrelated to resource requirements.

It started out in the same modal with the [CPU-features](https://github.com/openshift/openshift-origin-design/pull/375) but we extracted it since it is unrelated to CPU.
So, since the new section is named "Scheduling and Resources Requirements", should we pull it out of it and place it in the Details section above?
@jtomasek @glekne what do you think?

Reply:
@lizsurette 
I talked with Tomas about it.
He thinks it should be included in this section because:
1. it does related to CPU. Because of that, it should be located next to other CPU features. It will not make sense to break it from them
2. It does affect performance, so it is again related to "Resource Requirements".

Reply:
Makes sense to me, maybe we can update the documentation here to explain that to the reader?

Reply:
Added :)